README.md

```diff
@@ -437,7 +437,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [vue](https://github.com/tree-sitter-grammars/tree-sitter-vue) (maintained by @WhyNotHugo, @lucario387)
 - [x] [wgsl](https://github.com/szebniok/tree-sitter-wgsl) (maintained by @szebniok)
 - [x] [wgsl_bevy](https://github.com/theHamsta/tree-sitter-wgsl-bevy) (maintained by @theHamsta)
-- [x] [wing](https://github.com/winglang/wing) (experimental, maintained by @gshpychka, @MarkMcCulloh)
+- [x] [wing](https://github.com/winglang/tree-sitter-wing) (experimental, maintained by @gshpychka, @MarkMcCulloh)
```

Comment:
```suggestion
- [x] [wing](https://github.com/winglang/tree-sitter-wing) (maintained by @gshpychka, @MarkMcCulloh)
```

Reply:
It's currently flagged as experimental in parsers.lua, should this still be removed?

Reply:
Oh, sorry, I _meant_ in parsers.lua.